docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
+------------------
+
+.. versionadded:: 6.0.0
+
+The OME-TIFF specification supports multi-resolution images or pyramidal
+images where individual planes are stored at different levels of resolution.
+The intermediate downsampled image planes are called pyramidal levels,
```

Comment:
I don't think `intermediate` is necessary here.  It implies it doesn't include the smallest image at the end of the range, which it does.

`pyramid` is a better alternative to `pyramidal` here; it's concerning individual planes.